### PR TITLE
Update landingPage

### LINKED ISSUE #188: Design update - Margarita Landing Page
- [x] header
- [x] headline
- [x] graphql playground
- [x] showcase & info
- [x] footer

### PR DESCRIPTION
**Pull request by KarinaDavtyan**
Summary: update landingPage according to the new design

+ Also added `links_config.js`

+ But still has placeholder text and images
+ Some links missing
+ Not adaptive to mobile

Closes #188

### CODE REVIEW

File: apps/landingPage/src/components/demo.js [hunk 1]
```diff
@@ -1,61 +1,58 @@
...
 import { defaultTokens } from '@kiwicom/orbit-design-tokens';
-import { Link } from 'gatsby';
 import styled from 'styled-components';
+import { Instagram, Facebook } from '@kiwicom/orbit-components/lib/icons/';
 
-type Props = {|
-  +iframeWidth: number,
-  +iframeHeight: number,
-|};
+// $FlowFixMe - Cannot resolve module RelativeImageStub ( possible solution https://bit.ly/2BYfHRr )
```
Comment (tbergq) [inline apps/landingPage/src/components/demo.js]: This should no longer be necessary

File: apps/landingPage/src/components/header.js [hunk 2]
```diff
@@ -1,66 +1,56 @@
 // @flow
 
 import React from 'react';
-import { Link } from 'gatsby';
 import { Heading } from '@kiwicom/orbit-components/lib/';
 import styled from 'styled-components';
 
+// $FlowFixMe - Cannot resolve module RelativeImageStub ( possible solution https://bit.ly/2BYfHRr )
```
Comment (tbergq) [inline apps/landingPage/src/components/header.js]: Should not be necessary anymore

File: apps/landingPage/src/components/Demo.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+      <Heading type="title2">Learn more, visit Github</Heading>
+      <Button type="secondary" width={125}>
+        <LinkNoStyle href={github}>Link to Docs</LinkNoStyle>
+      </Button>
+    </ContainerTextRight>
+    <HeadingWrapper>
+      <Heading type="title1">More links to explore</Heading>
+    </HeadingWrapper>
+    <LinksWrapper>
+      {links.map(el => {
```
Comment (FilipMessa) [inline apps/landingPage/src/components/Demo.js]: Just cosmetic question.
The `el` is the abbreviation for an element, right? So should be this call `link` or something like that?

File: apps/landingPage/src/components/Headline.js [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+              PLACEHOLDER!!!!!You've had a look at Tequila, but you've preffered
+              yourTequila Tequila mixed with something else. Just grab
+              Margarita.You've had a look at Tequila, but you've always
+              preffered your Tequila mixed with something else.
+            </Text>
+          </WrapperText>
+          <WrapperButtons>
+            <Button>Get started</Button>
+            <WrapperRightButton>
+              <Button type="white">Link to Github</Button>
```
Comment (FilipMessa) [inline apps/landingPage/src/components/Headline.js]: Shouldn't be these `<Button>Get started</Button>` and ` <Button type="white">Link to Github</Button>` links?

File: apps/landingPage/src/pages/index.js [hunk 5]
```diff
@@ -3,15 +3,18 @@
...
+import Footer from '../components/Footer';
+import Demo from '../components/Demo';
+import Info from '../components/Info';
+import Headline from '../components/Headline';
+import Playground from '../components/Playground';
 
-const iframeWidth = 1200;
-const iframeHeight = 800;
+const windowWidth = window.innerWidth; //eslint-disable-line
+const windowHeight = window.innerHeight; //eslint-disable-line
```
Comment (FilipMessa) [inline apps/landingPage/src/pages/index.js]: I think that we can extend our the eslint setting about ` "browser": true`.

File: apps/landingPage/src/components/Headline.js (hunk 4 above)
Comment (KarinaDavtyan) [reply apps/landingPage/src/components/Headline.js]: Totally missed that. Thanks! Added links to margarita and github

File: apps/landingPage/src/components/Demo.js (hunk 3 above)
Comment (KarinaDavtyan) [reply apps/landingPage/src/components/Demo.js]: I was just avoiding using link.link but now i renamed to link.url everywhere

File: apps/landingPage/src/pages/index.js (hunk 5 above)
Comment (KarinaDavtyan) [reply apps/landingPage/src/pages/index.js]: Ok! added "browser": true there

**Comment by FilipMessa** [comment]
just last thing, we have the issue with circle-ci as you know, so I checked all branch in PR on my local machine and run `yarn test-ci`, and after added 'browser: true' to eslint config there appeared one small error
screen shot 2019-02-07 at 11 12 43

**Pull request merged**